### DISCUSSION TITLE
it is posible to connect multiple vercel projects (from differents vercel teams) to one monorepository?

### DISCUSSION THREAD

**Discussion by giancarlosisasi** [vercel/community#459] [Help] [answered]
Hi there
I was wondering if it is possible to connect multiple vercel projects (from different vercel teams) to one monorepository. 🤔 
(I have made a little test and it doesn't work very well because only one of the projects is deployed in each commit)

**Comment by swarnava** [comment, marked as answer]
This should do the trick: https://vercel.com/knowledge/using-vercel-cli-for-custom-workflows. You need to  find your teamId and the projectId and then to deploy each site using same vercel.json.
